course_contents/1_intro/notes/01. Data Types.md

```diff
@@ -0,0 +1,166 @@
+### Variables
+- A variable is a symbolic type associated with a value.
```

Comment:
Would a `name` work better here? I don't know how to describe a variable otherwise as it's also a known term in math and it's pretty much mutually understood.